ci-operator/config/openshift/multiarch/README.md

```diff
@@ -0,0 +1,39 @@
+Multi-Payload Job Names
+=======================
+
+### What is the motivation?
+
+The multiarch team is migrating many of the single-architecture payload jobs to use the
+muli-architecture payload.  One of the difficulties when migrating these jobs is finding appropriately
+descriptive names for any of these new jobs so they do not conflict with existing job names,
+and reflect the nature of the cluster deployment.
+
+### Name Fields Breakdown
+
+Building upon the [Naming Your CI Jobs](https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/) documentation, the following pattern will be used for the multiarch
```

Comment:
Worth adding a link from naming-your-ci-jobs back to here as a reference?

Reply:
I'll add this as an item to work on in the near future (updating docs.ci to reflect information for multiarch and link back to this README, I mean)